deny.toml

```diff
@@ -252,6 +252,63 @@ deny = [
 
 # Certain crates/versions that will be skipped when doing duplicate detection.
 skip = [
+
+    # Requires updating slipped10 to newest sha2 others are dependents that will be updated
+    { crate = "sha2", reason = "temporary duplicate until upstream updates" },
```

Comment:
Do we plan to clean up all of this before the release? Or will some of this dependencies stay past the release?

Reply:
There are a lot of third party dependencies that we need to advocate to update. So my guess is that this would be an on going effort if the deduplication of crates is a priority.